### PR TITLE
Search and follow chart averaging

### PR DESCRIPTION
**Pull request by anna1901**

### CODE REVIEW

**Comment by anna1901** [comment]
[Implement averaging in the chart](https://trello.com/c/IbhLI7o8/618-implement-averaging-in-the-chart)

File: AirCasting/SearchAndFollow/CompleteScreen/Views/SearchAndFollowChartViewModel.swift [hunk 1]
```diff
@@ -19,13 +19,32 @@ class SearchAndFollowChartViewModel: ObservableObject {
...
+            }
+            
+            if buffer.isEmpty || hourIsAlreadyPresent(in: buffer.map({ $0.time }), date: measurement.time) {
+                buffer.append(measurement)
+                continue
+            }
+            
+            let average = (buffer.map { $0.value }.reduce(0, +)) / Double(buffer.count)
+            
+            times.append(buffer.last!.time.roundedUpToHour)
```
Comment (jankrzempek) [inline AirCasting/SearchAndFollow/CompleteScreen/Views/SearchAndFollowChartViewModel.swift:37]: The only thing is force unwrapping here, but after my long mind-analyze, I think it should be fine. Mentioning this only for the sake of having this in mind :)

**Review by jankrzempek — APPROVED** [review]
Seems fine 😎

File: AirCasting/SearchAndFollow/CompleteScreen/Views/SearchAndFollowChartViewModel.swift (hunk 1 above)
Comment (pawelgil) [reply AirCasting/SearchAndFollow/CompleteScreen/Views/SearchAndFollowChartViewModel.swift:37]: idk, for me it's ok, but on the edge. The code that guarantees that `buffer.last` won't ever be `nil` is like 4 lines up

**Review by pawelgil — APPROVED** [review]

**Pull request merged**